cfme/configure/settings.py

```diff
@@ -23,109 +25,173 @@ class TimeProfileAddForm(View):
+
+        return: It returns the object of the copied time_profile.
+        """
+
+        view = navigate_to(self.parent, 'All')
+        rows = view.time_profile_form.table
         for row in rows:
             if row.description.text == self.description:
                 row[0].check()
-        view.timeprofile_form.configuration.item_select('Copy selected Time Profile')
+        view.time_profile_form.configuration.item_select('Copy selected Time Profile')
         if name is not None:
-            new_timeprofile = Timeprofile(description=name, scope=self.scope)
-            changed = view.timeprofile_form.fill({
+            new_time_profile = self.parent.instantiate(description=name, scope=self.scope)
+            changed = view.time_profile_form.fill({
                 'description': name,
                 'scope': self.scope,
             })
         else:
-            new_timeprofile = Timeprofile(description="{} copy".format(self.description),
-                                          scope=self.scope)
-            changed = view.timeprofile_form.fill({
+            new_time_profile = self.parent.instantiate(
+                description="{} copy".format(self.description),
+                scope=self.scope
+            )
+            changed = view.time_profile_form.fill({
                 'description': "{} copy".format(self.description),
                 'scope': self.scope,
             })
 
         if changed:
-            view.timeprofile_form.save_button.click()
-        return new_timeprofile
+            view.time_profile_form.save_button.click()
+        return new_time_profile
+
+
+@attr.s
+class TimeProfileCollection(BaseCollection):
+
+    ENTITY = TimeProfile
+
+    def create(self, description, scope, days, hours, timezone, cancel=False):
+        """
+        Args:
+            description (str): It's the descriptive name of the time_profile.
+            scope: It's the option 'All User' or 'Current User' from dropdown.
+            days (bool): It's the option to switch on or switch off the days Bootstrap switch.
+            hours (bool): It's the option to swich on or switch off the hours Bootstrap switch.
+            timezone: It's the required Time Zone for the time_profile.
+            cancel (bool) : It's a flag used to cancel or not the create operation.
+
+        return: It returns the object of the newly created time_profile object.
+        """
+
+        time_profile = self.instantiate(description, scope, days, hours, timezone)
 
-    def delete(self):
         view = navigate_to(self, 'All')
-        rows = view.timeprofile_form.table
-        for row in rows:
-            if row.description.text == self.description:
-                row[0].check()
-        view.timeprofile_form.configuration.item_select("Delete selected "
-                                                        "Time Profiles", handle_alert=True)
+        view.time_profile_form.configuration.item_select('Add a new Time Profile')
```

Comment:
**Required**

This should be an 'Add' navigation destination registered against the TimeProfileCollection class.